_benchmark/reference/workloads/operations.md

```diff
@@ -0,0 +1,322 @@
+  }
+}
+```
+
+The following example creates a new index with all index settings specified in the operation body:
+
+```yml
+{
+  "name": "create-an-index",
+  "operation-type": "create-index",
+  "index": "people",
+  "body": {
+    "settings": {
+      "index.number_of_shards": 0
+    },
+    "mappings": {
+      "docs": {
+        "properties": {
+          "name": {
+            "type": "text"
+          }
+        }
+      }
+    }
+  }
+}
+```
+
+### Options
+
+Use the following options when creating all indexes from the `indices` section of a workload.
+
+Parameter | Required | Type | Description
+:--- | :--- | :--- | :---
+`settings` | No | Array |  Specifies additional index settings to be merged with the index settings specified in `indices` section of the workload.
+`request-params` | No | List of settings | Contains any request parameters allowed by the Create Index API. OpenSearch Benchmark does not attempt to serialize the parameters and pass them as is. 
+
+Use the following options when creating a single index in the operation.
+
+Parameter | Required | Type | Description
+:--- | :--- | :--- | :---
+`index` | Yes | String | The index name.
+`body` | No | Request body | The request body for the Create Index API. For more information, see [Create Index API](/api-reference/index-apis/create-index/)
+`request-params` | No | List of settings | Contains any request parameters allowed by the Create Index API. OpenSearch Benchmark does not attempt to serialize the parameters and pass them as is. 
+
+### Metadata
+
+The `create-index` operation returns the following metadata:
+
+`weight`: The number of indexes created by the operation.
+`unit`: Always `ops`, for the number of operations inside the workload.
+`success`: A Boolean indicating whether the operation has succeeded.
+
+## delete-index
+
+The `delete-index` operation runs the [Delete Index API](api-reference/index-apis/delete-index/). Like the [`create-index`](#create-index) operation, you can delete all indexes found in the `indices` section of the workload or delete one or more indexes based on the string passed in the `index` setting.
+
+### Usage
+
+The following example deletes all indexes found in `indices` section of the workload:
+
+```yml
+{
+  "name": "delete-all-indices",
+  "operation-type": "delete-index"
+}
+```
+
+The following example deletes all `logs_*` indexes:
+
+```yml
+{
+  "name": "delete-logs",
+  "operation-type": "delete-index",
+  "index": "logs-*",
+  "only-if-exists": false,
+  "request-params": {
+    "expand_wildcards": "all",
+    "allow_no_indices": "true",
+    "ignore_unavailable": "true"
+  }
+}
+```
+
+### Options
+
+Use the following options when deleting all indexes indicated in the `indices` section of the workload.
+
+Parameter | Required | Type | Description
+:--- | :--- | :--- | :---
+`only-if-exists` | No | Boolean | Decides whether an index should be deleted only if the index exists. Default is `true`.
```

Comment:
This is an odd one because the setting itself is almost unnecessary. You wouldn't be able to delete something if it hadn't existed previously. 

```suggestion
`only-if-exists` | No | Boolean | Decides whether an index should be deleted when the index exists. Default is `true`.
```

Reply:
Does that help? Open to suggestions on this one.

Reply:
"Decides whether an existing index should be deleted"?